source/fundamentals/connection.txt

```diff
@@ -230,6 +239,12 @@ URI to specify the behavior of the client.
        server documentation on the
        :manual:`w Option </reference/write-concern/#w-option>`.
 
+   * - **directConnection**
+     - boolean
+     - ``false``
+     - Specifies whether to force dispatch **all** operations to the server
```

Comment:
**questions:**

1. Would it make more sense to say "funnels all operations"? This makes it clear it could create a bottleneck or other undesirable condition.
2. Would it not be `host`?

Reply:
Definitely agree with host. Funnel is incorrect here, it actually dispatches to the host.